### PR TITLE
feat(component): Updated Website Components

### PR DESCRIPTION
**Pull request by thefiend**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,11 +19,14 @@
...
     "@ledgerhq/hw-transport-u2f": "4.48.0",
     "@ledgerhq/web3-subprovider": "4.48.0",
+    "@zeit/next-sass": "^1.0.1",
+    "classnames": "^2.2.6",
     "debug": "^4.1.1",
     "eslint-plugin-react": "^7.7.0",
     "lodash": "^4.17.5",
     "next": "^8.0.0",
     "next-redux-wrapper": "^1.3.5",
+    "node-sass": "^4.12.0",
```
Comment (Nebulis) [inline package.json]: why sass, whats the need ?

Comment (thefiend) [reply package.json]: Importing of .scss into components

Comment (Nebulis) [reply package.json]: let me rephrase => why not using plain css/emotion/postcss :)
